proposals/NNNN-Avoid_custom_button_subscription_when_HMI_does_not_support.md

```diff
@@ -0,0 +1,75 @@
+
+# Avoid Custom button subscription in case HMI incompatibility
+
+* Proposal: [SDL-NNNN](NNNN-Avoid_custom_button_subscription_when_HMI_does_not_support.md)
+* Author: [Getmanets Irina](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+In case HMI does not support `CUSTOM_BUTTON`, SDL should not try to subscribe an application to it.
+
+## Motivation
+
+SDL does not check hmi_capabilities before subscription to `CUSTOM_BUTTON`, so it tries to subscribe each registered application to `CUSTOM_BUTTON` even if it is unsupported by HMI.
+
+SDL should check hmi_capabilities and subscribe application to `CUSTOM_BUTTON` if it is **supported by HMI only**.
+
+## Proposed solution
+
+SDL should check hmi_capabilities for `CUSTOM_BUTTON` by the start and save the support state for `CUSTOM_BUTTON` internally.
```

Comment:
Is additional **the** needed: `by the start and save **the** support state`?

If not change to:
SDL should check hmi_capabilities for `CUSTOM_BUTTON` by the start and save support state for `CUSTOM_BUTTON` internally.

Reply:
@jordynmackool we will remove the extra "the" and resubmit.